docs/howto_guides/api_gui.md

```diff
@@ -0,0 +1,161 @@
+    - `insert`: inserts a new value
+    - `remove item`: removes the value from a specified attribute.
+    - `update`: replaces the existing value with a new value 
+    - `delete`: removes the attribute entirely from the metadata document
+3. `attribute`: the attribute (or field/slot) that will be updated (e.g. `name` or `ecosystem_category`, etc.)
+4. `value`: the new value that will be inserted or that will replace the old value.
+
+Please note that if changes are made to multivalued attributes that have a "structured" value, e.g. [air_temp_regm](https://microbiomedata.github.io/nmdc-schema/air_temp_regm/) has a range of [TextValue](https://microbiomedata.github.io/nmdc-schema/TextValue/), set the `value` to a variable, then set the `id` of the next line to the variable with the `value` set to what will get populated. See [example](https://github.com/microbiomedata/nmdc-runtime/issues/284#issuecomment-1686825159). For more information, please see [Authoring Changesheets](https://microbiomedata.github.io/nmdc-runtime/howto-guides/author-changesheets/)<br/>
+<br/>
+
+![metadata post json validate](../_static/images/howto_guides/api_gui/metadata_post_json_validate.png)
+Metadata may also be validated in a JSON format, which can be posted in the body of the request and validated against the NMDC schema using the `POST /metatdata/json:validate` endpoint before final submission to the portal.<br/>
+<br/>
+
+![metadata post json validate urls file](../_static/images/howto_guides/api_gui/metadata_post_validate_urls_file.png)
+A text file of urls that point to a JSON object may be supplied using the `POST /metadata/json:validate_urls_file` endpoint. This is helpful for validation of very large JSON metadata or if a user has a link to a JSON file but is not in an environment where it is convenient to download the file and then upload it to validate.<br/>
+<br/>
+
+![metadata get nmdcshema version](../_static/images/howto_guides/api_gui/metadata_get_nmdcschema_version.png)
+To view the [NMDC Schema](https://microbiomedata.github.io/nmdc-schema/) version the database is currently using, try executing the `GET /nmdcschema/version` endpoint<br/>
+<br/>
+
+![metadata get collection stats](../_static/images/howto_guides/api_gui/metadata_get_collection_stats.png)
+To get the NMDC Database collection statistics, like the total count of records in a collection or the size of the collection, try executing the `GET /nmdcschema/collection_stats` endpoint<br/>
+<br/>
+
+![metadata get collection name](../_static/images/howto_guides/api_gui/metadata_get_collection_name.png)
+The `GET /nmdcschema/{collection_name}` endpoint is a general purpose way to retrieve metadata about a specified collection given user-provided filter and projection criteria. Please see the [Collection Names](https://microbiomedata.github.io/nmdc-schema/Database/) that may be retrieved. Please note that only one collection set may be retrieved at a time.<br/>
```

Comment:
When I read this sentence, I wondered what a collection set was. Based on my interpretation of the rest of the paragraph, here's a version of the sentence that doesn't use that term:

```diff
- Please note that only one collection set may be retrieved at a time.
+ Please note that metadata may only be retrieved about one collection at a time.
```

Reply:
Thanks! I replaced this sentence with your suggestion: [6af0528](https://github.com/microbiomedata/NMDC_documentation/pull/27/commits/6af0528dcecd180368a851e7e571c45fbc41ece7)